src/libtrx/include/libtrx/game/creature/const.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "../const.h"
```

Comment:
Let's avoid pulling broad dependencies in modules like this.

Reply:
Thanks. I was thinking this was needed for `WALL_L` but it compiles without it, didn't check that before.